server/templates/events.html

```diff
@@ -7,7 +7,7 @@ <h3>Events</h3>
         <table class="table table-bordered table-striped events-table" style="width: 100%">
             <thead>
               <tr>
-                <th>Sensor</th>
+                <th>Camera</th>
```

Comment:
I also just went ahead and changed the column headings on the events page from Sensor to Camera. We aren't using any sensors so this makes more sense.